src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -91,25 +91,45 @@ bool mpu6500SpiDetect(void)
     return false;
 }
 
+void mpu6500SpiAccInit(acc_t *acc)
+{
+    mpu6500AccInit(acc);
+}
+
 bool mpu6500SpiAccDetect(acc_t *acc)
 {
     if (mpuDetectionResult.sensor != MPU_65xx_SPI) {
         return false;
     }
 
-    acc->init = mpu6500AccInit;
+    acc->init = mpu6500SpiAccInit;
     acc->read = mpuAccRead;
 
     return true;
 }
 
+void mpu6500SpiGyroInit(uint8_t lpf)
+{
+    spiSetDivisor(MPU6500_SPI_INSTANCE, SPI_CLOCK_SLOW);
+    delayMicroseconds(1);
+
+    mpu6500GyroInit(lpf);
+
+    // Disable Primary I2C Interface
+    mpu6500WriteRegister(MPU_RA_USER_CTRL, MPU6500_BIT_I2C_IF_DIS);
```

Comment:
this i probably not nesessary and the most important part were delays in mpu6500GyroInit . without them RAW_RDY_EN was not enabled